### PR TITLE
Made the index.html, register.html, login.html use base template

### PR DESCRIPTION
**Pull request by elenatorrey**
fixes issue #86

## Changes in this PR.

- [X] base.html is set up for index, login and register pages
- [X] had to remove the {% javascript 'libraries' %} line because apparently the pipeline compressor isn't configured completely and it was causing an error when testing 

## Testing this PR.

1. flake8
2. ./runtests.sh
3. docker-compose up

### Expected Output.

Ran 11 tests in 0.388s

OK
Destroying test database for alias 'default'...

@osuosl/devs

### CODE REVIEW

**Comment by elenatorrey** [comment]
@subnomo @Kennric @leian7 @alxngyn

File: streamwebs_frontend/streamwebs/templates/streamwebs/base.html [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="utf-8" />
     <title>{% block title %}Streamwebs{% endblock %}</title>
     <link rel="stylesheet" type="text/css" href="{% static 'streamwebs/style.css' %}" />
     <link rel="shortcut icon" href="{% static 'streamwebs/images/favicon.ico' %}" />
-    {% javascript 'libraries' %}
+    {% load pipeline %}
```
Comment (LyonesGamer) [inline streamwebs_frontend/streamwebs/templates/streamwebs/base.html]: You need to keep the `{% javascript %}` tag here. `{% load pipeline %}` just tells Pipeline to find the static files and collect them, `{% javascript 'libraries' %} causes it to actually inject the <script> tags.

File: streamwebs_frontend/streamwebs/templates/streamwebs/index.html [hunk 2]
```diff
@@ -1,11 +1,8 @@
...
-<html>
-    <head>
-        <title>StreamWebs</title>
-    </head>
-
-    <body>
-        <h1>StreamWebs</h1>
+{% extends 'streamwebs/base.html' %}
+{% load staticfiles %}
+{% load pipeline %}
```
Comment (Kennric) [inline streamwebs_frontend/streamwebs/templates/streamwebs/index.html]: I think staticfiles and pipeline includes can be moved into base, they will be used in all of the child templates

File: streamwebs_frontend/streamwebs/templates/streamwebs/base.html (hunk 1 above)
Comment (alxngyn) [inline streamwebs_frontend/streamwebs/templates/streamwebs/base.html]: should we ignore/remove pipeline completely and just import the JS libraries by <script> tags?

Comment (elenatorrey) [inline streamwebs_frontend/streamwebs/templates/streamwebs/base.html]: @alxngyn we ended up removing the load pipeline tags

**Comment by LyonesGamer** [comment]
:+1:

File: streamwebs_frontend/streamwebs/templates/streamwebs/sites.html [hunk 3]
```diff
@@ -1,8 +1,6 @@
 {% extends "streamwebs/base.html" %}
-
 {% load staticfiles %}
-
-{% block title %}Sites - Streamwebs{% endblock %}
+{% block title %}Sites -{% endblock %}
 
 {% block scripts %}
```
Comment (subnomo) [inline streamwebs_frontend/streamwebs/templates/streamwebs/sites.html]: You need to include jQuery as well, since we aren't using the pipline anymore.

**Comment by subnomo** [comment]
Looks good. :+1:

File: streamwebs_frontend/streamwebs/templates/streamwebs/base.html [hunk 4]
```diff
@@ -1,19 +1,21 @@
...
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="utf-8" />
-    <title>{% block title %}Streamwebs{% endblock %}</title>
+    <title>{% block title %}{% endblock %} StreamWebs</title>
     <link rel="stylesheet" type="text/css" href="{% static 'streamwebs/style.css' %}" />
     <link rel="shortcut icon" href="{% static 'streamwebs/images/favicon.ico' %}" />
-    {% javascript 'libraries' %}
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```
Comment (LyonesGamer) [inline streamwebs_frontend/streamwebs/templates/streamwebs/base.html]: We already have a static version installed via bower. Run `bower install`, then change this to:

    <script src="{% static 'streamwebs/bower_components/jquery/dist/jquery.js' %}"></script>

**Comment by elenatorrey** [comment]
@leian7 @LyonesGamer  I had docker issues and I couldn't run the tests on this, can you please do that for me?

**Comment by subnomo** [comment]
Lookin' good :+1:

**Comment by Kennric** [comment]
Looks good. +1

**Pull request merged**